docs/articles/utilities/serilog.md

```diff
@@ -35,9 +43,9 @@ var log = Context.GetLogger();
 log.Info("The value is {0}", counter);
 ```
 
-To log using Serilog syntax you need to use the `SerilogLogMessageFormatter`:
+Or alternatively
 ```csharp
-var log = Context.GetLogger(new SerilogLogMessageFormatter());
```

Comment:
Why remove this? Isn't that necessary to get the semantic Serilog-style formatting to work?

Reply:
Hi, I found when I ran the examples, that it did not seem to make a difference in output whether the formatter was included or not.  I felt if it was not necessary or no longer necessary, it was probably best to remove from the example.  It seems allot of this structuring of output is handled in serilog - https://github.com/serilog/serilog/wiki/Structured-Data#default-behaviour - some of this page seems to apply to 2.6.0 but the following seems to apply for 2.5.0

```csharp
Log.Information("The value is {counter}", counter);
```

Reply:
funnily, the following works too

```csharp

private void Handle(int count)
{
	var log = Context.GetLogger();
	log.Info("Count is {a} {b} {c}", count, 2, true);
}
```
> 2018-01-08 18:23:12 INF Count is 1 2 True

Reply:
I see.

Reply:
Correct. The SerilogLogMessageFormatter is no longer needed.